tooltester/src/main/resources/JenkinsReadme.md

```diff
@@ -0,0 +1,21 @@
+Master Setup:
```

Comment:
I've already made changes to the README to incorporate markdown, but will be doing a second pull request after this.